### ISSUE TITLE
newer classes equal or not - footnote reference

### ISSUE THREAD

**Issue by lethargosapatheia** [jokteur/python_communism#5]
I love the idea, I love the rationale, I love everything about this project. I wish all devs had your vision and transformed all programming languages into equal languages. I abhor discrimination and I don't understand why some languages such as php are supposed to be "inferior". I'm looking forward to the day the first operating system is written completely in php and a little bit of html 3 and 4.

That aside, regarding your second footnote:
> The communist revolution does not prevent future classes to be equal to all others. Classes declared after the communism.revolution() will not follow the communism rule. ↩

Isn't this supposed to be something to the extent of "does not prevent future classes to be superior to others"? If newer classes are equal to all others, then that's consistent with the Communist Revolution, isn't it? (by the way, please use uppercases out of respect).

**Comment by HacKanCuBa** [comment]
Newer classes will not be affected by revolution, thus won't be equal unless a new revolution happens afterwards.
This means that we either maintain a permanent revolution, or classes will be different!

**Comment by mononobi** [comment]
This is why all communist regimes are constantly on revolutionary order.

**Comment by jokteur** [comment]
Everything is working as (not) intended.

**Comment by lethargosapatheia** [comment]
> This is why all communist regimes are constantly on revolutionary order.

I agree with that and I find this today more relevant than ever, given the technological revolutions we're seeing on a daily basis (apple deciding to revolutionarily remove the jack, for example, or I don't know, apple's touchbar, or, just another random example, apple deciding that you should have the keyboard replaced every 6 months to a year for free). We see the same revolution happening in art, of course. I haven't seen a non-revolutionary contemporary piece of art maybe for 20 years I think.



> Newer classes will not be affected by revolution, thus won't be equal unless a new revolution happens afterwards. This means that we either maintain a permanent revolution, or classes will be different!

I am happy to dogmatically accept the reasoning behind this. Otherwise how would the Revolution even take place if everyone where the question its motives?

**Comment by derekriemer** [comment]
I think we could use threading to make a compromise solution. We simply make the revolution occur perpetually. As soon as one revolution ends, a new is started.
